tsconfig.json

```diff
@@ -58,5 +58,6 @@
     /* Experimental Options */
     // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
+    "skipLibCheck": true
```

Comment:
What does this actually do?

Reply:
From the docs:

> Skip type checking of all declaration files (*.d.ts).

Since we are in a TS project, this skips checking that any dependencies' types are correct. This mostly fixes a persistent problem I have where builds break for me with errors like this:

```
node_modules/typescript/lib/lib.dom.d.ts:16532:11 - error TS2430: Interface 'WebGL2RenderingContext' incorrectly extends interface 'WebGL2RenderingContextBase'.
  Types of property 'clearBufferfv' are incompatible.
    Type '(buffer: number, drawbuffer: number, values: ArrayLike<number> | Float32Array, srcOffset?: number | undefined) => void' is not assignable to type '{ (buffer: number, drawbuffer: number, values: Float32List, srcOffset?: number | undefined): void; (buffer: number, drawbuffer: number, values: Iterable<number>, srcOffset?: number | undefined): void; }'.
      Types of parameters 'values' and 'values' are incompatible.
        Type 'Iterable<number>' is not assignable to type 'ArrayLike<number> | Float32Array'.
          Type 'Iterable<number>' is missing the following properties from type 'Float32Array': BYTES_PER_ELEMENT, buffer, byteLength, byteOffset, and 25 more.

16532 interface WebGL2RenderingContext extends WebGL2RenderingContextBase, WebGL2RenderingContextOverloads, WebGLRenderingContextBase {
```

Though if you've seen this before and fixed it in a different way I am very interested!

Reply:
I don't think I've ever seen that before. So I'm fine with that. i was just curious